.goreleaser.yml

```diff
@@ -0,0 +1,23 @@
+project_name: falco-exporter
+before:
+  hooks:
+    - go mod tidy
+builds:
+  - id: "falco-exporter"
+    goos:
+    - linux
+    goarch:
+    - amd64
+    - 386
+    main: ./cmd/falco-exporter
+    env:
+      - GO111MODULE=on
```

Comment:
Is this necessary?

Reply:
Actually, I don't believe 386 is really needed, although I'm not sure if someone might use it.
If we would not support that, I can remove it.

Reply:
not really, removed!
Thanks :hugs: